region/tourian/main/Mother Brain Room.json

```diff
@@ -656,6 +656,34 @@
       ],
       "devNote": "This technically requires f_KilledZebetites1 to not be set, but there is no loss of access, because there is no reason to kill any Zebs without being able to kill them all."
     },
+    {
+      "link": [2, 3],
+      "name": "X-Mode Shinecharge, R-Mode Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "h_CrystalFlashForReserveEnergy",
+        "f_KilledZebetites1",
+        "f_KilledZebetites2",
+        "f_KilledZebetites3",
+        "f_KilledZebetites4",
+        {"not": "f_MotherBrainGlassBroken"},
+        "canRiskPermanentLossOfAccess",
+        "Morph",
+        "canUseFrozenEnemies",
+        "canXMode",
+        {"spikeHits": 1},
+        "h_shinechargeMaxRunway",
+        {"autoReserveTrigger": {"maxReserveEnergy": 95}},
+        "canRModeSparkInterrupt"
+      ],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "Bounce off of a frozen Rinka and onto the corner to enter Spike X-Mode. Gain shinecharge, then use any Rinka to interrupt."
```

Comment:
- Quickly freeze all active rinkas at the same time to CF without dying.
- Mention the MB platform should be treated as a spike.